src/Makefile

```diff
@@ -123,6 +125,7 @@ cpp_common_test_test_LDFLAGS := -L../usr/lib \
                                 -ldl \
                                 -lpthread \
                                 -lboost_regex \
+                                -lboost_system \
```

Comment:
Not sure why we require this additional dependency?
Is it from something you've included?

Reply:
Likely something unintentionally included, ill investigate and remove it

Reply:
This is required by the SasService in cpp-common